api/src/api/routes/v1/brla.route.ts

```diff
@@ -24,7 +25,6 @@ router.route('/triggerOfframp').post(validateBrlaTriggerOfframpInput, brlaContro
 
 router.route('/createSubaccount').post(validataSubaccountCreation, brlaController.createSubaccount);
 
-router.route('/startKYC2').post(brlaController.startKYC2);
-
+router.route('/startKYC2').post(startKYC2Validator, brlaController.startKYC2);
```

Comment:
How about the following name change to align with the other namings?

```suggestion
router.route('/startKYC2').post(validateStartKyc2, brlaController.startKYC2);
```